code/modules/surgery/advanced/pacification.dm

```diff
@@ -23,12 +35,24 @@
 	implements = list(
 		TOOL_HEMOSTAT = 100,
 		TOOL_SCREWDRIVER = 35,
-		/obj/item/pen = 15)
+		/obj/item/pen = 15,
+	)
 	time = 40
 	preop_sound = 'sound/surgery/hemostat1.ogg'
 	success_sound = 'sound/surgery/hemostat1.ogg'
 	failure_sound = 'sound/surgery/organ2.ogg'
 
+/datum/surgery_step/pacify/mechanic
+	name = "delete aggression programming (multitool)"
```

Comment:
we really need to automate these tool names inside the surgery step names (not anything you need to do in this PR)